lib/src/test/java/space/taran/arklib/user/score/RootScoreStorageTests.kt

```diff
@@ -0,0 +1,60 @@
+package space.taran.arklib.user.score
+
+import dev.arkbuilders.arklib.user.score.RootScoreStorage
+import dev.arkbuilders.arklib.user.score.ScoreMonoid
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.GlobalScope
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import space.taran.arklib.randomResourceId
+import space.taran.arklib.randomScore
+import java.nio.file.Path
+
+class RootScoreStorageTests {
+
+    private val scope: CoroutineScope by lazy {
+        GlobalScope
+    }
+
+    private val pathMock: Path by lazy {
+        mockk()
+    }
+
+    private lateinit var rootScoreStorage: RootScoreStorage
```

Comment:
Each test case should use its own instance of `RootScoreStorage`. This way, each test case would be isolated.